app/containers/Send/Send.jsx

```diff
@@ -139,7 +139,10 @@ export default class Send extends React.Component<Props, State> {
         )
         const price = prices[asset]
 
-        const totalBalanceWorth = multiplyNumber(balance, price)
+        const totalBalanceWorth = multiplyNumber(
+          Number(balance).toFixed(15),
```

Comment:
mul throws an error if the string or number is greater than 15 significant digits - this fixes that bug

```
export const multiplyNumber = (x: string | number, y: string | number) =>
  toBigNumber(x)
    .mul(y)
    .toNumber()
```